src/World.java

```diff
@@ -92,10 +88,34 @@ else if(i==1) {
         return theCitiesOnMap;
     }
 
+    //NOTE: For now, keeping it simple.  Just random.  Might want to re-balance payout later.
+    public CargoOrder getCargoOrder() {
+        //Chose a random product, city and payout.
+        Random random = new Random();
+        Product chosenProduct = Product.valueOf(Product.getRandom().name());
+        City chosenCity = cities[random.nextInt(cities.length)];
+        int chosenPayout = random.nextInt(20,50);
+
+        //Create and return the CargoOrder.
+        CargoOrder cargoOrder = new CargoOrder(chosenProduct, chosenCity, chosenPayout);
+        return cargoOrder;
+    }
+
     public Terrain getTerrain(int x, int y) {
         return map[x][y];
     }
 
+    public void runTests() {
+        Log.debug("");
+        Log.debug("Condensed city info:");
+        for (City c : cities) {
+            Log.debug(c.toString());
+        }
+        Log.debug("");
+        Log.debug("Test Cargo Order Generation:");
+        for (int i=0; i<15; i++) Log.debug(getCargoOrder().toString());
```

Comment:
I have strong opinions on always bracketing for loops. I've seen too many bugs come from non-bracketed ones. I really wish Java enforced this as part of the language.

Reply:
OK.